scss/_accordion.scss

```diff
@@ -0,0 +1,146 @@
+//
+// Base styles
+//
+
+.accordion-button {
+  @include font-size($font-size-base);
+  display: flex;
+  align-items: baseline;
+  width: 100%;
+  padding: $accordion-button-padding-y $accordion-button-padding-x;
+  color: $accordion-button-color;
+  text-align: left;
+  background-color: $accordion-button-bg;
+  border: $accordion-border-width solid $accordion-border-color;
+  border-bottom: none;
+  @include border-radius(0);
+
+  &:not(.collapsed) {
+    color: $accordion-button-active-color !important; // stylelint-disable-line declaration-no-important
```

Comment:
Any reason why these `!important`s where added? Seems to work fine without?

Reply:
Aha, probably because of the zebra stripes? Maybe then just use the `:not()` construction there too? This way, people will still be able to override the behaviour with utilities.

Reply:
Yes this was one of the things I was planning on refactoring, will look at it hopefully this weekend :D